C2GUILauncher/src/Views/MainWindow.xaml.cs

```diff
@@ -18,13 +22,53 @@ public partial class MainWindow : Window {
 
         private readonly ModManager ModManager;
 
+        public int ShowInstallRequestFor(string TargetDir, InstallationType InstallType)

```

Comment:
rather than returning ints to indicate the result of an install request, I think it'd be better to have an Enum